### ISSUE TITLE
Does five come as a jQuery plugin?

### ISSUE THREAD

**Issue by uzquiano** [jackdcrawford/five#27]
This is useless unless it works with jQuery.  Anyone who is a hacker uses jQuery and not JavaScript because it is broken and doesn't work on IE.

High $.five()

**Comment by lpar** [comment]
Yes, my first thought was that it really ought to require jQuery, because you can't do anything in JavaScript without jQuery.
